examples/mockups/index.html

```diff
@@ -0,0 +1,329 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="/dist/css/site.css" />
+    <script src="/dist/js/vendor.js"></script>
+    <script src="/dist/js/site.js"></script>
+  </head>
+  <body>
+
+    <!-- Top Menu 
+    ----------------------------------- -->
+    <div class="ui menu topnav fixed borderless">
+      <div class="ui container">
+
+        <a class="header item" href="">
+          <div class="ui image readthedocs-logo"><img src="./images/logo-wordmark-dark.svg"/></div>
+        </a>
+
+        <a class="item active" href="">Product</a><!-- page -->
+        <a class="item" href="">Pricing</a><!-- page -->
+        <a class="item" href="">Company</a><!-- page -->
+        <a class="item" href="">Resources</a><!-- Docs, Blog, Tutorials, Developers (page), Github  -->
+    
+        <div class="right menu">
+          <a class="item" href="">Support</a><!-- Bug Report (Github), Feature Request (Github), User Support (page), Premium Support (link to bookmark in the pricing page), Contact us (page) -->
+          <div class="item">
+            <a class="ui button primary">
+              <!-- Note: The account login or Sign up can be dealt with when arriving to the application -->
+              Go to Application
+            </a>
+          </div>
+        </div>
+    
+      </div>
+    </div>
+    <!-- end Top Menu 
+    ----------------------------------- -->
+
+
+    <!-- Contents 
+    ----------------------------------- -->
+
+    <!-- Callout -->
+    <section class="ui vertical segment very padded" id="callout">
+
+      <div class="ui container">
+        <div class="ui grid centered middle aligned">
+
+          <div class="row">
+
+            <div class="column seven wide computer nine wide tablet sixteen wide mobile">
+              <h1 class="ui header">
+                Continuous documentation for technical projects
+              </h1>
+              <p class="lead text">
+                Make documentation part of your project workflow with simplified processes that automate building, versioning and hosting of your docs.
+              </p>
+              
+              <div class="group wrapper">
```

Comment:
`.group.wrapper` adds a margin bottom to the buttons to account for the responsive flow.
**i.e.** it creates a vertical spacing for when the buttons pile vertically on small screens.
https://github.com/readthedocs/site-community/blob/0f4c9b83ebd22a7c488ba1055cd692116ad24453/src/sui/themes/rtd-site/elements/button.overrides#L98-L113

Reply:
Doesn't a two column grid, stacking on mobile, work the same way?

Reply:
Doesn't a two column grid, stacking on mobile, accomplish this?

Reply:
I'm not sure how to accomplish this and maintain the required margin. Especially because margin between two buttons in a group will be different from a margin I'll want for different elements. But it can probably be done... I'll check.

Reply:
I ended up not adding columns - as I want the two buttons to lay side by side - and just removed the `.group.wrapper` for now. We can come back to this later when we are defining the responsive behaviour.

It's probably worth noting that SUI uses custom a custom `docs.css` on their docs, to add margin bottom to their buttons, conditionally on a `@media` query.